service-base/pom.xml

```diff
@@ -112,10 +108,32 @@
       <artifactId>quarkus-smallrye-health</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.jaegertracing</groupId>
-      <artifactId>jaeger-core</artifactId>
-      <scope>compile</scope>
-      <optional>true</optional>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-opentelemetry</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-opentelemetry-exporter-otlp</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.code.findbugs</groupId>
+          <artifactId>jsr305</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <!-- JAX-RS API used by quarkus-opentelemetry; using Jakarta implementation here instead of org.jboss.spec.javax.ws.rs:jboss-jaxrs-api_2.1_spec used by quarkus-rest-client -->
+      <groupId>jakarta.ws.rs</groupId>
+      <artifactId>jakarta.ws.rs-api</artifactId>
+    </dependency>
+    <!-- adding support for jaeger propagator -->
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-extension-trace-propagators</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-opentracing-shim</artifactId>
```

Comment:
FMPOV this should be in services/base-quarkus/pom.xml and adapters/base-quarkus/pom.xml as the tracing implementation is generally only required by the service and adapter implementations, isn't it?

Reply:
I've adapted the poms.
`quarkus-opentelemetry` and `opentelemetry-opentracing-shim` are still needed here though (used in `SamplerProducer`, `SamplingPrioritySampler` for example).